packages/graBing/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname="graBing"
+pkgver=9.bb8b08b
+pkgrel=1
+pkgdesc="Counts all the hostnames for an IP adress"
+arch=('x86_64' 'i686')
```

Comment:
I can't see why not **`'i686' 'x86_64' 'armv6h' 'armv7h' 'aarch64'`**  or simply **`any`**.

Reply:
I don't know much about it, so I just put the arch that i tried this and work.